### DISCUSSION TITLE
Can I create separate "workspaces" for home and work?

### DISCUSSION THREAD

**Discussion by Centauri27** [standardnotes/forum#3428] [General]
I'm wondering if Standard Notes supports separate "workspaces". For example, I'd like to use it to track my work and personal notes. But I don't want to see all the notes and tags from my work cluttering up my personal notes. 

This sounds like a simple request. Though I've been using Standard Notes for a while for personal notes, I'm now thinking of moving my work notes into Standard Notes as well. 

Is this as simple as creating two top level notes for "Personal" and "Work" and then build out the trees beneath them?

**Comment by effieeee** [comment]
Hi @Centauri27, unfortunately we don't have a function that compartmentalizes your notes into a set for work, and another for your personal ones. We do have a "Workspaces" function, but this is meant for using **multiple accounts" on the app, rather than creating a subset of your notes. However, you can still try to make use of this in your case, by creating a separate account for your work notes, then use the Workspaces function as described in the link below, so that you can seamlessly switch between your two accounts:

https://standardnotes.com/help/82/can-i-sign-into-multiple-accounts-at-the-same-time

**Comment by Centauri27** [comment]
Hi. This idea might work. But I paid for a premium account (that's
supposedly linked to my email address?). Won't creating another account
with a different email put me back into the free tier? Or can I create
separate accounts that's linked to my paid account?

**Comment by effieeee** [comment]
I see, though is your account on a **Professional Plan** in particular? If so, you can [share your subscription](https://standardnotes.com/help/86/share-subscription-with-another-account) with your work account!

**Comment by Centauri27** [comment]
No, I have the "Productivity" subscription, not "Professional". That means
I'm out of luck and can't use this workspace workaround?

**Reply by effieeee** [reply]
I see, I’m sorry to hear that, though yes, that would mean that you cannot share your subscription with another account that’s signed into a workspace 😢
